wbooks/src/app/screens/Login/index.tsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import { useForm, Controller } from 'react-hook-form';
 import { useNavigation } from '@react-navigation/native';
+import { useDispatch } from 'react-redux';
+import actionCreators from '@redux/auth/actions';
 import { ImageBackground, Text, TouchableOpacity } from 'react-native';
 import Form from '@screens/Login/components/Form';
 import bcInicio from '@assets/General/bc_inicio.png';
 
 import styles from './styles';
 
+interface DataForm {
+  email: string;
+  password: string;
+}
+
 const Login = () => {
   const { control, handleSubmit, errors } = useForm();
   const navigation = useNavigation();
+  const dispatch = useDispatch();
 
-  const onSubmit = () => navigation.navigate('Library');
+  const onSubmit = (data: DataForm) => {
```

Comment:
extract email and password from data.

```suggestion
  const onSubmit = (data: DataForm) => {
```
```suggestion
  const onSubmit = ({ email, password }: DataForm) => {
```

Reply:
Done!